packages/create-cloudflare/templates/pre-existing/c3.ts

```diff
@@ -0,0 +1,91 @@
+import type { C3Context } from "types";
+import { chooseAccount } from "../../src/common";
+import { processArgument } from "helpers/args";
+import { cp, mkdtemp, readdir, rm } from "fs/promises";
+import { join } from "path";
+import { tmpdir } from "os";
+import { runCommand } from "helpers/command";
+import { detectPackageManager } from "helpers/packages";
+import { brandColor, dim } from "@cloudflare/cli/colors";
+
+const config = {
+	configVersion: 1,
+	id: "pre-existing",
+	displayName: "Pre-existing Worker (from Dashboard)",
+	platform: "workers",
+	hidden: true,
+	languages: ["js"],
+	copyFiles: {
+		path: "../hello-world/js",
```

Comment:
Would it be simpler to have a "pre-existing" template, which has no wrangler.toml nor src files; this would avoid having to delete them.

Reply:
That's a fantastic point 😄